airflow/bin/cli.py

```diff
@@ -1457,7 +1457,7 @@ class CLIFactory(object):
         'subdir': Arg(
             ("-sd", "--subdir"),
             "File location or directory from which to look for the dag",
-            default=settings.DAGS_FOLDER),
+            default="[AIRFLOW_HOME]/dags"),
```

Comment:
Unfortunatley changing this default actually changes the value that will be in `args.subdir` - so this change would break a lot of things.

It might be possible to change the `$AIRFLOW_HOME` when rendering the docs, or to change the display of this param?

@kaxil How are the docs built when render

Reply:
You're right. Didn't realize this. I would leave this to @kaxil to decide how he would like to fix this (please feel free to close this PR should you're going to fix this separately).

But we do need to fix this. `Default: /Users/kaxil/airflow/dags` in the documentation is very confusing and misleading to doc readers.